src/All.php

```diff
@@ -0,0 +1,155 @@
+<?php namespace Leean\Acf;
+
+/**
+ * Class to provide helpers for getting a all ACF fields for an entity.
+ */
+class All
+{
+	/**
+	 * Is the ACF plugin active.
+	 *
+	 * @return bool
+	 */
+	public static function is_active() {
+		return function_exists( 'get_field_object' );
+	}
+
+	/**
+	 * Apply the default transforms if any exist for this field type.
+	 *
+	 * @param array $field The field.
+	 * @return mixed
+	 */
+	private static function apply_default_transform( $field ) {
+		$class = '\\' . __NAMESPACE__ . '\\Transforms\\' . str_replace( '_', '', ucwords( $field['type'], '_' ) );
+
+		if ( method_exists( $class, 'apply' ) ) {
+			return call_user_func( [ $class, 'apply' ], $field );
+		}
+
+		return $field['value'];
+	}
+
+	/**
+	 * Get all field values.
+	 *
+	 * @param int $target The target object.
+	 * @return array
+	 */
+	private static function get_fields( $target = 0 ) {
+		$data = [];
+
+		if ( self::is_active() ) {
+			$fields = get_field_objects( $target );
+
+			if ( $fields ) {
+				foreach ( $fields as $field_name => $field ) {
```

Comment:
I don't think so since if we do that we are creating a pure function that recieves a single item instead, and yes I think is how function should work having and input generates an output.